spellbook/models/balances/ethereum/erc721/balances_ethereum_erc721_hour.sql

```diff
@@ -22,14 +22,13 @@ with
     lead(hour, 1, now()) OVER (PARTITION BY token_address, wallet_address ORDER BY hour) AS next_hour
     FROM {{ ref('transfers_ethereum_erc721_rolling_hour') }})
 
-SELECT
+SELECT distinct
     'ethereum' as blockchain,
     d.hour,
     b.wallet_address,
     b.token_address,
     b.tokenId,
-    nft_tokens.name as collection,
-    nft_tokens.category as category
```

Comment:
Is deleting the whole category column the only/best way ? It seems like it's an easy/temporary fix but it would still be cool to include categories in erc721 balances. Wouldn't distinct fix it ?

Reply:
no, just a temp fix. we should try to do a group by list aggregation but I need to figure out how to do it in sparksql.

Reply:
Ok, if you think it'll take a long time to find the right fix we can push this but it made me a bit hesitant at first because it kind of feels like it wouldn't be too much work to get it fixed without excluding categories

Reply:
I'll still approve I don't want it to block you or anything